src/applications/ivc-champva/10-7959f-1/config/prefillTransformer.js

```diff
@@ -5,6 +5,7 @@ const prefillTransformer = (pages, formData, metadata) => {
       ...formData,
       veteranFullName: formData.veteranFullName,
       veteranAddress: formData.veteranAddress,
+      veteranHomßeAddress: formData.physicalAddress,
```

Comment:
Looks like "veteranHomßeAddress" has an extra character between the 'm' and the 'e'

Reply:
LOL yeah that was me attempting to save...badly. I changed it to residential address though because I remembered you mentioned that.